firebase-components/firebase-dynamic-module-support/firebase-dynamic-module-support.gradle.kts

```diff
@@ -22,6 +22,7 @@ firebaseLibrary {
   testLab.enabled = false
   publishSources = true
   publishJavadoc = false
+  publishReleaseNotes = false
```

Comment:
https://firebase.google.com/support/release-notes/android?_gl=1*ab5rhc*_up*MQ..*_ga*NzA5NzI4OS4xNzA3OTQyNzMw*_ga_CW55HF8NVT*MTcwNzk0MjczMC4xLjAuMTcwNzk0MjczMC4wLjAuMA..#dynamic-feature-modules-support_v16-0-0-beta03

Reply:
Were these being made manually? Why is there nothing in the changelog file?

Reply:
Yes, because:


Last Updated Date | 9/1/2022 ([maven](https://maven.google.com/web/index.html?q=dynamic#com.google.firebase:firebase-dynamic-module-support:16.0.0-beta03))


So it's being a while since we made a release of it